tech_foul_app/update.js

```diff
@@ -31,10 +31,9 @@ mongoose
         previous_xp_advantage: '7',
         previous_gold_advantage: '8'
       }]
-    });
+    })
     matchone.save(function (error, document) {
-      if (error) console.error(error);
-      console.log(document);
-    });
+      if (error) console.error(error)
+      console.log(document)
+    })
```

Comment:
You can delete this file, as I was just using it to populate the MongoDB with dummy data.

Reply:
We'll do a pruning after the pull request